oe_theme.theme

```diff
@@ -846,3 +846,17 @@ function oe_theme_preprocess_responsive_image_formatter(&$variables) {
   $variables['responsive_image']['#attributes']['class'][] = 'ecl-image';
   $variables['responsive_image']['#attributes']['class'][] = 'ecl-image--fluid';
 }
+
+/**
+ * Implements template_preprocess_preprocess_block().
+ *
+ * For preprocessing default mode of document media type.
+ */
+function oe_theme_preprocess_block(&$variables) {
+  $variables['title_attributes'] += [
```

Comment:
This code only works if `title_attributes` doesn't have classes.

Try:
```
$variables =["title_attributes" => ['class' => 'myclass']];

 $variables['title_attributes'] += [
    'class' => [
      'ecl-heading',
      'ecl-heading--h2',
    ],
  ];

var_dump($variables);
```

Result: 
```
array(1) {
  ["title_attributes"]=>
  array(1) {
    ["class"]=>
    string(7) "myclass"
  }
}
```